docs/pages/careers/staff-ui-engineer-base-ui.md

```diff
@@ -0,0 +1,93 @@
+# Staff UI Engineer — Base UI
+
+<p class="description">Research, build, document, and ship high-quality, unstyled UI components with a focus on a11y.</p>
+
+## Details of the role
+
+- **Location**: Remote (preference for UTC-6 to UTC+5).
+- **Type of work**: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501#494af1f358794028beb4b7697b5d3102)).
+- **Level**: [IC5 or above](https://mui-org.notion.site/Levelling-at-MUI-5c30f9bfe65149d697f346447cef9db1).
+
+## About MUI
```

Comment:
Can we follow the template: https://github.com/mui/material-ui/blob/master/docs/pages/careers/ROLE_TEMPLATE.md? If you want to propose updates to the template, I propose opening a pull request separately. What do you think?